package.json

```diff
@@ -20,7 +20,13 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "version": "conventional-changelog -p angular -i CHANGELOG.md -t v -s && git add CHANGELOG.md"
```

Comment:
I think it's good to have a `version` script, but I am not sure about the `&& git add CHANGELOG.md` part, ideally the git add should be done by CI